bitnami/external-dns/templates/secret.yaml

```diff
@@ -48,7 +48,7 @@ data:
   infoblox_wapi_password: {{ .Values.infoblox.wapiPassword | b64enc | quote }}
   {{- end }}
   {{- if eq .Values.provider "pdns" }}
-  pdns_api_key: {{ .Values.pdns.apiKey . | b64enc | quote }}
+  pdns_api_key: {{ .Values.pdns.apiKey | b64enc | quote }}
```

Comment:
Great! This could be the issue you were facing when using `.Values.pdns.apiKey` alone.